frontend/src/assets/style.css

```
@@ -11,26 +11,30 @@
-      'Apple Color Emoji',
-      'Segoe UI Emoji',
-      'Segoe UI Symbol',
-      'Noto Color Emoji';
+    font-family: var(--font-sans);
+    font-size: var(--text-body);
+    letter-spacing: var(--tracking-normal);
     line-height: 1.5;
   }
 }
 
 @theme inline {
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)